tips.json

```diff
@@ -7,6 +7,9 @@
 	}, {
 		"title": "Search change by content",
 		"tip": "git log -S'<a term in the source>'"
+	}, {
+		"title": "Show changes over time for specific file",
+		"tip": "git log -p <file_name>"
```

Comment:
This wasn't covered!? 😲

Reply:
Didn't get that!

Reply:
As in till now we didn't have that tip in our list.